jobs/narre/stats/JEVS_NARRE_STATS

```diff
@@ -7,11 +7,10 @@ set -xa
 #######################################################
 #######################################################
 
-export cyc=${cyc:-"00"}
-export cycle=${cycle:-t${cyc}z}
+export cycle=t${cyc}z
 
 export pid=$$
-export DATA=${DATA:-${DATAROOT}/${jobid:?}}
+export DATA=${DATA:-${DATAROOT:?}}/${jobid:?}
 mkdir -p $DATA
```

Comment:
@BinbinZhou-NOAA This change should not have ben made.  Please revert to:

`export DATA=${DATA:-${DATAROOT:?}/${jobid:?}}`

Thanks!